pages/glossary/distinguishing-styles.md

```diff
@@ -0,0 +1,11 @@
+--
+title: Distinguishing Styles
```

Comment:
Instead of having this definition use three other definitions, I think this should be a single definition. I don't think we currently have a use for those three as separate definitions. Even if at some point in the future we need it, we can split it out then.

Reply:
Even if don't have another use for these defintions, it is much easier to read the definitions if they are split like this. Don't you think that it makes it worth to have them split?

Reply:
I also prefer the split up version, but it's mainly a personal choice.